userspace/libsinsp/sinsp.h

```diff
@@ -519,6 +519,12 @@ class SINSP_PUBLIC sinsp : public capture_stats_source, public wmi_handle_source
 
 	libsinsp::event_processor* m_external_event_processor;
 
+	sinsp_threadinfo* build_threadinfo()
+    {
+        return m_external_event_processor ? m_external_event_processor->build_threadinfo(this)
```

Comment:
See overall review comment for my thoughts on this method

Reply:
you missed the big one....namely that if an external processor is registered without overriding the build_threadinfo method, then this will invoke the default implementation which will invoke this again in a tight loop